src/pystatis/table.py

```diff
@@ -142,6 +148,19 @@ def get_data(
             "format": "ffcsv",
```

Comment:
you can directly add the parameters here

Reply:
Updated